src/tests/example.spec.js

```diff
@@ -28,3 +28,83 @@ test.describe('Saucedemo app basic tests', () => {
         await expect(app.shoppingCart.cartItems).not.toBeAttached();
     });
 });
+
+test.describe('Saucedemo Unit 10 tests', () => {
+    test('sorting z-a', async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    ) => {
+        await app.login.navigate();
+        await app.login.performLogin('standard_user', 'secret_sauce');       
```

Comment:
Lets have these 2 lines moved to `test.beforeEach` hook, so we dont need to duplicate this code.